examples/js/controls/TransformControls.js

```diff
@@ -1353,7 +1350,8 @@ THREE.TransformControlsGizmo = function () {
 
 				} else {
 
-					handle.material.opacity *= 0.05;
+					handle.material.opacity *= 0.25;
```

Comment:
I also changed the amount of dimming when other axis is highlighted. Before, it was pretty much invisible at 0.05 opacity. Now you can see the axes but faintly.